src/connectors/store.py

```
@@ -1,22 +1,111 @@
+    
+    class Config:
+        arbitrary_types_allowed = True
+    
+    @classmethod
+    def load_local(cls, path: str, k: int = 256) -> "BM25Retriever":
+        """Carga índice BM25 desde disco."""
+        instance = cls(k=k)
+        instance.index = bm25s.BM25.load(path, load_corpus=False)
+        
+        with open(os.path.join(path, "documents.pkl"), "rb") as f:
+            instance.documents = pickle.load(f)
```

Ruff (0.14.10)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)